packages/expo-image-manipulator/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import { NativeModulesProxy } from 'expo-core';
```

Comment:
I think `index.ts` -> `ImageManipulator.ts` may be a good idea here, that's what is advised here https://github.com/expo/expo/blob/master/guides/Migrating%20Universal%20Modules%20to%20TypeScript.md#change-import